src/scss/fonts.scss

```diff
@@ -0,0 +1,238 @@
+/* poppins-300 - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 300;
+  src: url("../assets/font/poppins/poppins-v5-latin-300.eot"); /* IE9 Compat Modes */
+  src: local("Poppins Light"), local("Poppins-Light"),
+    url("../assets/font/poppins/poppins-v5-latin-300.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("../assets/font/poppins/poppins-v5-latin-300.woff2")
+      format("woff2"),
+    /* Super Modern Browsers */
+      url("../assets/font/poppins/poppins-v5-latin-300.woff") format("woff"),
+    /* Modern Browsers */ url("../assets/font/poppins/poppins-v5-latin-300.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("../assets/font/poppins/poppins-v5-latin-300.svg#Poppins")
+      format("svg"); /* Legacy iOS */
+}
+/* poppins-300italic - latin */
+@font-face {
+  font-family: "Poppins";

```

Comment:
Remove all these font-faces.

Reply:
removed

Reply:
Please remove the font file in your src/assets folder.